WebUI/Data/DataAccess/IUserService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ConestogaInsidersClub.Data.Models;
+using System.Collections.Generic;
+
+namespace ConestogaInsidersClub.Data.DataAccess
+{
+    public interface IUserService
+    {
+        ApplicationUser GetUser(string userName);
+        List<ApplicationUser> GetUsers();
```

Comment:
This looks good, although I would make it a habit to have DB access functions asynchronous. If you leave it as is, nothing bad will happen, but imagine if extracting a user from the DB takes a few seconds: your whole app would be frozen and not responsive as your main thread is synchronously waiting for the extraction to complete. You can read more about it [here](https://docs.microsoft.com/en-us/ef/core/miscellaneous/async).